### PR TITLE
Added a forecasting recipe

### PR DESCRIPTION
**Pull request by guilhermebodin**
#170 I think we could benefit from some simple plot recipes, no more searching how to make a proper ribbon.

Maybe we can even define more recipes for scenarios and components in the future.

Look what have I done in a feel lines 
```julia
using StateSpaceModels, CSV, DataFrames, Plots
nile = CSV.read(StateSpaceModels.NILE, DataFrame)
model = LocalLevel(nile.flow)
fit!(model)
forec = forecast(model, 24)
plot(model, forec)

air_passengers = CSV.read(StateSpaceModels.AIR_PASSENGERS, DataFrame)
log_air_passengers = log.(air_passengers.passengers)
model = BasicStructural(log_air_passengers, 12)
fit!(model)
forec = forecast(model, 24)
plot(model, forec; legend = :topleft)
```

Captura de Tela 2020-12-13 às 20 07 36
Captura de Tela 2020-12-13 às 20 05 56

### CODE REVIEW

**Comment by raphaelsaavedra** [comment]
I'm not familiar with `RecipesBase`, what's the difference between this and simply extending `plot`?

**Comment by guilhermebodin** [comment]
You don't need to have Plots.jl as a dependency.

File: src/visualization/forecasting.jl [hunk 1]
```diff
@@ -0,0 +1,42 @@
+RecipesBase.@recipe function f(model::StateSpaceModel, forec::Forecast)
+    if !isa(model.system.y, Vector)
+        error("This plot recipe currently works for univariate models only.")
+    end
+    dists = Vector{Distribution}(undef, length(forec.expected_value))
+    for d in eachindex(dists)
+        dists[d] = StateSpaceModels.Distributions.Normal(forec.expected_value[d][1], sqrt(forec.covariance[d][1]))
```
Comment (raphaelsaavedra) [inline src/visualization/forecasting.jl:7]: Are the prefixes necessary here?

**Comment by raphaelsaavedra** [comment]
Other than a small detail which I'm not sure is necessary, looks good to me!

File: src/visualization/forecasting.jl (hunk 1 above)
Comment (guilhermebodin) [reply src/visualization/forecasting.jl:7]: Nop

**Comment by guilhermebodin** [comment]
I should test with this https://github.com/JuliaPhysics/Measurements.jl/blob/master/test/plots.jl

**Pull request merged**